portal-frontend/src/app/features/edit-application/parcel-entry/parcel-entry.component.scss

```diff
@@ -0,0 +1,39 @@
+@import '../../../../styles/functions.scss';
```

Comment:
So do I get it right that since there are no designs, this is just a plane form for now, correct?

Reply:
Yes, its just a form that only does lookup, nothing is saved to the application yet.